### ISSUE TITLE
Cleanup `RWA007-A` auto-line config params

### ISSUE THREAD

**Issue by amusingaxl** [makerdao/spells-goerli#189]
The auto-line for `RWA007-A` [has been disabled](https://github.com/makerdao/spells-goerli/blob/6ef52322370bd219368e7941db0080f2181a65e6/archive/2023-03-28-DssSpell/DssSpell.sol#L123), however the old params in the config file [were left there](https://github.com/makerdao/spells-goerli/blob/6ef52322370bd219368e7941db0080f2181a65e6/src/test/config.sol#L1376-L1378):

![image](https://user-images.githubusercontent.com/112016538/233717610-b5a03cce-d41f-4568-9ab0-9654135c5a82.png)

The issue is that this does not reflect the current state of the `MCD_IAM_AUTO_LINE` contract:

![image](https://user-images.githubusercontent.com/112016538/233717967-8e19124b-127f-467d-9910-2e7fe2fd0605.png)

The test didn't catch this because they [simply ignore the values if `aL_enabled` is `false`](https://github.com/makerdao/spells-goerli/blob/6ef52322370bd219368e7941db0080f2181a65e6/src/DssSpell.t.base.sol#L486-L489).

We probably need to fix the test to make it enforce the auto-line parameters are cleaned up when an ilk auto-line is removed.

**Comment by The-Arbiter** [comment]
I agree with this, perhaps as part of a spell cleanup PR we can change this from 
`aL_gap: 100 * MILLION` 
to 
`aL_gap: 0 // Previously 100 * MILLION`
